world.js

```diff
@@ -781,8 +824,8 @@ const _connectRoom = async (roomName, worldURL) => {
 
 const objects = [];
 world.getObjects = () => objects;
-world.addObject = (contentId, parentId = null, position = new THREE.Vector3(), quaternion = new THREE.Quaternion(), options = {}) => {
-  state.transact(() => {
+world.addObject = async (contentId, parentId = null, position = new THREE.Vector3(), quaternion = new THREE.Quaternion(), options = {}) => {
```

Comment:
Heads up, this should probably not be `async` in the end.

[`contentId` need not be a token at all](https://github.com/webaverse/app/blob/a67a805227758078ddd329b488316d39007536a7/world.js#L858); it could also be a url in which case we can't look up monetization on our end.

It's also probably not a critical error for the monetization flow to fail and throw (we should still load the object).

Reply:
- e4ebb02 removes the added async and moves it to existing token backend call on trackedobjectadd event
- 9cd290c checks for pointer before pushing
- 0e6414a adds checking for [valid monetization pointer synax](https://paymentpointers.org/#syntax).